lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/ProcessJourneyEventHandler.java

```diff
@@ -76,6 +77,8 @@ public class ProcessJourneyEventHandler
     private static final String NEXT_EVENT = "next";
     private static final StepResponse BUILD_CLIENT_OAUTH_RESPONSE =
             new ProcessStepResponse(BUILD_CLIENT_OAUTH_RESPONSE_EVENT, null);
+    private static final StepResponse CROSS_BROWSER_PROBLEM_PAGE_RESPONSE =
+            new PageStepResponse(CROSS_BROWSER_PROBLEM_PAGE_EVENT, null, null);
```

Comment:
You're using the `CROSS_BROWSER_PROBLEM_PAGE_EVENT` string for both the journey event and also the page id.

Not a big deal, but it's unnecessary for them to be equal and results in slightly odd looking code
- here, you create a page response containing an event id
- in the test you assert that the page id is equal to an event id

Would it make more sense to define the two strings separately (even if they happen to have the same value)?